cmd/drand-cli/cli.go

```diff
@@ -911,7 +932,7 @@ func contextToConfig(c *cli.Context) *core.Config {
 	if c.Bool("tls-disable") {
 		opts = append(opts, core.WithInsecure())
 		if c.IsSet("tls-cert") || c.IsSet("tls-key") {
-			panic("option 'tls-disable' used with 'tls-cert' or 'tls-key': combination is not valid")
+			return nil, fmt.Errorf("option 'tls-disable' used with 'tls-cert' or 'tls-key': combination is not valid")
```

Comment:
So instead of returning an error for every command now (that adds like 4lines * number of command) we can also do this check before running the command, there is a `Before` item in the Cli struct that allows this.

Reply:
I see the point, but here we are parsing c *cli.Context to *core.Config. Before requires a function which returns only an error.

Reply:
We only need to check consistency of the flags `if c.IsSet("tls-cert") || c.IsSet("tls-key")`  we can return the config still in this function but at least we know the flags are correct because we checked them before already.
 example: https://github.com/urfave/cli/blob/master/docs/v2/manual.md#full-api-example

Reply:
Could you check again?